stdlib/stdlib.ispc

```diff
@@ -1044,10 +1044,37 @@ __declspec(safe, cost1) static inline uniform double select(uniform bool cond, u
 
 __declspec(safe) static inline uniform int16 reduce_add(int8 x) { return __reduce_add_int8(__mask ? x : (int8)0); }
 
+__declspec(safe) static inline uniform int8 reduce_min(int8 v) {
+    // Set values for non-running lanes to the maximum int8 value so
+    // they don't affect the result.
+    int8 int8_max = 0x7F;
+    return __reduce_min_int8(__mask ? v : int8_max);
```

Comment:
I think we'd better use `INT8_MAX` here and in similar places, it should be defined for `stdlib.ispc` from here https://github.com/ispc/ispc/blob/ee408969519b859d065c27fca9adcc9b1e5cd4f8/stdlib/include/core.isph#L160

Reply:
Updated